platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -71,6 +71,15 @@
 	final int SCREEN_SENSOR_PORTRAIT = 5;
 	final int SCREEN_SENSOR = 6;
 
+	public static final int KEYBOARD_TYPE_DEFAULT = 0;
+	public static final int KEYBOARD_TYPE_MULTILINE = 1;
+	public static final int KEYBOARD_TYPE_NUMBER = 2;
+	public static final int KEYBOARD_TYPE_NUMBER_DECIMAL = 3;
+	public static final int KEYBOARD_TYPE_PHONE = 4;
+	public static final int KEYBOARD_TYPE_EMAIL_ADDRESS = 5;
+	public static final int KEYBOARD_TYPE_PASSWORD = 6;
+	public static final int KEYBOARD_TYPE_URL = 7;
```

Comment:
Let's move these constants to `GodotEditText`.

Also include a comment to remind to keep them in sync with the native versions.

Reply:
Will do.